src/AbstractMappingTransformer.php

```diff
@@ -35,21 +36,37 @@ public function __construct($accessor = null)
      * @param array $mapping The mapping to use to transform input array
      * @param array $data The input array
      * @return array The output array
-     * @throws OptimusException
+     * @throws OptimusException|JsonOptimusException
```

Comment:
` JsonOptimusException ` extends the `OptimusException ` class so useless addition

Reply:
You right